extension.js

```diff
@@ -557,7 +559,8 @@ const ClipboardIndicator = Lang.Class({
             ExtensionUtils.openPrefs();
         } else {
             Util.spawn([
-                "gnome-shell-extension-prefs",
+                "gnome-extensions",
+                "prefs",
```

Comment:
This has nothing to do with clipboard listener

Reply:
Without this you are not able to ope the setting dialog. For testing the change it is necessary.

Reply:
Yep. But should not be in PR commit. You could had you 'lister' branch (re)based after that change. You other PR have this lines changed also : accepting one will cause conflict on the other.

Reply:
I think this change is related to this PR.

Reply:
Why did you make this change?